digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/masstransitstop/oracle/queries.scala

```diff
@@ -237,6 +237,14 @@ object Queries {
     """.as[Int].list
   }
 
+  def getMunicipalitiesWithoutAhvenanmaa: Seq[Int] = {
+    var municipalitiesOfAhvenanmaa = List(35, 43, 60, 62, 65, 76, 170, 295, 318, 417, 438, 478, 736, 766, 771, 941)
```

Comment:
Ahvenanmaa road maintainer id is 0 -> maybe use road maintainer id instead? (if municipalities change the maintainer id should remain)